shared/util.js

```diff
@@ -832,7 +449,7 @@ const UIStrings = {
   /** Label indicating that Lighthouse throttled the page using custom throttling settings. */
   runtimeCustom: 'Custom throttling',
 };
-Util.UIStrings = UIStrings;
+Util.UIStrings = {...UIStrings};
```

Comment:
no idea why but was getting this error without making this change:

```
Property 'UIStrings' does not exist on type 'typeof Util'. Did you mean 'strings'?
```

Reply:
What's the point of having this now? `UIStrings` has a named export in this file.